test/segment-loader.test.js

```diff
@@ -233,7 +229,35 @@ QUnit.module('SegmentLoader: M2TS', function(hooks) {
       loader.mimeType(this.mimeType);
       loader.load();
 
-      this.startTime.returns(11);
+      mp4probe.timescale = function() {
```

Comment:
If you overwrite functions on an import like this, you should save the original functions to a variable and restore them at the end the test, or they could impact over tests in the module.

Reply:
No longer relevant since the tests have been moved to mux: https://github.com/videojs/mux.js/pull/304